src/components/pages/SupportPage/RenderSupportPageModal.js

```diff
@@ -10,7 +10,7 @@ function RenderSupportPage({ success }) {
   const [modalVisible, setModalVisible] = useState(false);
 
   //access the parents information in the reducer so we can pre-populate the form with their name & email.
-  const userInfo = useSelector(state => state.parent);
+  // const userInfo = useSelector(state => state.parent);
```

Comment:
Was just wondering what's going on here? Might be best to uncomment the code or remove it entirely?

Reply:
I'm actually a little confused by that, I don't recall messing with anything on the support page, unless this happened when I pulled from main

Reply:
Nevermind, I remember why those were commented out but uncommented them. I thought they were messing up a test and as per one of the members, I was commenting those out, but that was not the problem so I fixed it again